lib/model.js

```diff
@@ -1792,6 +1794,13 @@ module.exports = (function() {
     }
   };
 
+  Model.prototype.__getDefaultTimestamp = function(attr) {
+    if (!!this.rawAttributes[attr] && !!this.rawAttributes[attr].defaultValue) {
+      return Utils.toDefaultValue(this.rawAttributes[attr].defaultValue);
+    }
+    return undefined;
+  };
+
```

Comment:
I'd actually remove the old __getTimestamp (It's only used twice right now) and only always __getDefaultTimestamp in combination with Utils.now(this.sequelize.options.dialect). What do you think?